.github/workflows/plugins-test.yaml

```diff
@@ -464,4 +464,30 @@ jobs:
       - name: Build the Docker image
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run kafka 0.11.0.0-2.3.0 (16)
-        run: bash test/plugin/run.sh kafka-scenario
\ No newline at end of file
+        run: bash test/plugin/run.sh kafka-scenario
+
+Mysql:
```

Comment:
This is not a correct format, This is a YAML.

Reply:
The indentation is not correct IMO, syntax looks good